py/janitor/site/simple.py

```diff
@@ -705,12 +705,18 @@ async def main_async(argv=None):
     public_runner = web.AppRunner(public_app)
     await private_runner.setup()
     await public_runner.setup()
+
     site = web.TCPSite(private_runner, args.host, port=args.port)
     await site.start()
-    logging.info("Listening on %s:%s", args.host, args.port)
+    logging.info("Private aiohttp listening on %s:%s", args.host, args.port)
+
     site = web.TCPSite(public_runner, args.host, port=args.public_port)
     await site.start()
-    logging.info("Listening on %s:%s", args.host, args.public_port)
+    logging.info("Public aiohttp listening on %s:%s", args.host, args.public_port)
```

Comment:
The aiohttp bit is an implementation detail, not relevant here.

Reply:
...tweaked the wording once again.